common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -39,6 +39,10 @@ class ProjectCard extends React.PureComponent<Props, State> {
       }),
       showModal: false,
     };
+    // iFrameResizer will startup after the page is rendered, so we need to rerener if it does
+    if(!window.iFrameResizer) window.iFrameResizer={}
+    if(!window.iFrameResizer.onInParent) window.iFrameResizer.onInParent=[]
+    window.iFrameResizer.onInParent.push(this.onIframeResizer.bind(this))
```

Comment:
This should live in a helper function or wrapper component. 
Also, don't forget line semicolons!

Reply:
This was an interesting challenge. I created common/components/common/IframeResizerInParent.jsx a wrapper component with static helper functions that cleanly factored this out.